communications/newsletters/newsletter_26_June2021.md

```diff
@@ -0,0 +1,175 @@
+- [Presenting Posters and Conference Talks](https://the-turing-way.netlify.app/communication/presentations.html)
+- [Social Media for Research Communications](https://the-turing-way.netlify.app/communication/social-media.html)
+
+## Contributor in Focus: Marta Mangiarulo
+
+![Picture of Marta. She is a white woman, with light brown hair. She is wearing a black sleeveless top and sunglasses. She is holding a red bottle with the soap solution and blowing soap bubbles]images/2021-06-marta.png)
+
+Marta Mangiarulo is a Research Assistant and Teaching Fellow in Psychology at the University of Leicester. 
+Her research interests involve probabilistic inference, social hypothesis testing, and reasoning biases. 
+She is also passionate about science communication and research dissemination and interested in replicability, open science issues, and the interface between cognitive and social aspects in social psychology. As the two times Book Dash attendee, Marta reflects on her participation:
+> In the **November 2020 book dash**, I used GitHub for the first time! I started familiarising myself with the GitHub environment and learned how to collaborate through it to provide valuable contributions to the project. 
+> My work during this book dash had mostly been individual, but in the May 2021 book dash, I was very happy to start where I had left off. 
+> I collaborated with others, made relevant contributions and I have a clear idea of where I would like to go next which is much more than what I was hoping for! 
+> I loved taking part in the Book Dash, it is such a rewarding experience above and beyond its material outputs.
+
+## Relevant news and resources
+
+### Open Life Science Application Call is Open!
+
+[Open Life Science](https://openlifesci.org/) (OLS) is a **16-week mentoring and training program** that upskills researchers in open and reproducible research and empowers them to become open science ambassadors in their communities. 
+We have opened a call for applications for the next cohort, OLS-4, which will take place from September 2021 to January 2022. The deadline to submit your application is **15 July 2021**. Visit https://openlifesci.org for details.
+OLS has received the [Turing Online Training Grant](https://www.applieddatascience.qmul.ac.uk/news/3817/the-turing-online-training-call-funding-opportunity), to specifically train and mentor the Turing and The Turing Way members under the collaboration name **OLS for Turing** (see [promotion pack](https://docs.google.com/document/d/e/2PACX-1vTH0hdIN8EeWdH_v-_LW0Yiy5cLD9BeATLwvzBuUrlooFqZxJ_DIX7tIcA1H2WfXR3j3RJ34wgxnfzs/pub)). We will also offer microgrants to our participants to ensure that they can equitably participate in the program.
+
+![OLS flyer that provides a timeline of the programme from September 2021 to January 2021](images/2021-06-ols.png)
+
+*Open Life Science programme details and timeline for the fourth cohort.*
```

Comment:
Is this to be added or removed?

Reply:
It should be added. Let me check if the image renders